buddyboss/buddyboss-platform
# PROD-893 Fixed # 1807 : Members Count is still showing when the member profile type is hidden

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1807.

### How to test the changes in this Pull Request:

1. Go to BuddyBoss > Profile > Profile Types > Add new Profile type
2. Set the settings like this https://prnt.sc/vmmrbw
3. Assign a User the newly created profile type from the Extended Profile settings https://prnt.sc/vmmv27
4. Search with the username in the Search field from the front end and **hit Enter**. You will see the members panel in the sidebar and the count. But it has nothing https://i.imgur.com/d2zOjED.png 

### Proof Screenshots or Video

https://www.loom.com/share/e4b65d6fef3d425fba9b64a2bd39b84c

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
